src/panels/config/areas/ha-config-area-page.ts

```diff
@@ -215,7 +215,12 @@ class HaConfigAreaPage extends LitElement {
       <hass-subpage
         .hass=${this.hass}
         .narrow=${this.narrow}
-        .header=${area.name}
+        .header=${html`${area.icon
+          ? html`<ha-icon
+              .icon=${area.icon}
+              style="margin-inline-end: 8px;"
+            ></ha-icon>`
+          : nothing}${area.name}`}
```

Comment:
Use class instead of inline style.

Reply:
Can't use class here as it is passed to another component (should've used the slot instead...)

Reply:
Oh right!